app/views/govuk_wrapper.scala.html

```diff
@@ -39,20 +39,6 @@
     linkElem = Some(linksElement),
     headScripts = None)
     <meta name="format-detection" content="telephone=no" />
-    <script type="text/javascript">
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-                    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-
-            ga('create', '@Some(appConfig.analyticsToken)', 'auto');
-
-
-
-            ga('send', 'pageview', { 'anonymizeIp': true });
-    </script>
-    <script src='@routes.Assets.at("javascripts/tabs.js")'></script>
 }
 
 @linksElement = {
```

Comment:
As per reference PRs we can remove this one also

`@footerLinks = {}`

Reply:
Sure, taken that out now.